README.md

```diff
@@ -1,2 +1,78 @@
 # server-template
 A quickstart template of a server-side application powered by Spine.
+
+## Prerequisites
+
+* Install JDK 7 or higher.
```

Comment:
You use Java 8 in the code, not majorly, but still. I guess we should either change the above line, or adjust the code.